web/src/views/Dashboard.tsx

```diff
@@ -74,9 +74,10 @@ const Dashboard: React.FC = () => {
     verifyProjects();
   }, 1000 * 30);
 
+
   return (
     <>
-      {userRoles.includes('administrator') && <ProjectRequests profileDetails={profileDetails} />}
```

Comment:
I am not sure if we want to let them see all those requests, we can check this with Olena!